seed/seeds.txt

```diff
@@ -1,3 +1,3 @@
 https://dbpedia.org/resource/New_York
```

Comment:
Maybe we should remove these "default" URIs?

1. They are DBpedia URIs although the DBpedia itself shouldn't be crawled.
2. It might be better to have either an empty file or no file at all. Otherwise, people will start to crawl the example URIs while they wanted to crawl something else :thinking:

Reply:
done